core/src/main/java/org/ehcache/config/writebehind/WriteBehindConfigurationBuilder.java

```diff
@@ -58,19 +58,31 @@ public WriteBehindConfigurationBuilder segment(int concurrency) {
     return this;
   }
   
-  public WriteBehindConfigurationBuilder coalesce() {
+  public WriteBehindConfigurationBuilder enableCoalescing() {
     this.writeCoalescing = true;
     
     return this;
   }
   
-  public WriteBehindConfigurationBuilder batch(int batchSize) {
+  public WriteBehindConfigurationBuilder disableCoalescing() {
+    this.writeCoalescing = false;
+    
+    return this;
+  }
+  
+  public WriteBehindConfigurationBuilder enableBatching(int batchSize) {
```

Comment:
I would revert the change with regards to batching. `batch(1)` was a good way of saying no watching. It would require the logic to be improved though - including refusing values 0 or below.